src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -28,6 +28,7 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
   const [allSchedules, setAllSchedules] = useState<Schedule[]>([]);
   const [selectedSchedule, setSelectedSchedule] = useState<Schedule | null>(null);
   const [selectedCourse, setSelectedCourse] = useState<CourseEvent | null>(null);
+  const [accordionState, setAccordionState] = useState<boolean>(true);
```

Comment:
Rather than needing a separate useEffect to get the stored state, you can just check localStorage here:
```suggestion
  const [accordionState, setAccordionState] = useState<boolean>(localStorage.getItem('accordionState') === 'true' ?? true);
```

Reply:
Could we come up with a clearer name for this state variable? As it is currently, I'm unsure whether `accordionState === true` means the accordion is open or closed. Maybe `isScheduleOpen` would be better?
```suggestion
  const [isScheduleOpen, setIsScheduleOpen] = useState<boolean>(localStorage.getItem('accordionState') === 'true' ?? true);
```